### DISCUSSION TITLE
Specify font size in "add_axes_at_origin"

### DISCUSSION THREAD

**Discussion by SunilAnandatheertha** [nschloe/pygmsh#528] [Q&A]
Is there a way we can specify the font size in "add_axes_at_origin" ?

**Comment by nschloe** [comment]
There's no such thing as `add_axes_at_origin` in pygmsh. Wrong repo?

**Reply by SunilAnandatheertha** [reply]
oh ohh, sorry.. was referring to Plotter() under pyvista ! Wrong repo indeed. But, if you know about it, please let me know, much appreciated.  In any case, pygmsh is awesome. Thanks for this.
